app/src/renderer/vuex/modules/user.js

```diff
@@ -13,7 +13,7 @@ export default ({ commit, node }) => {
   }
 
   const emptyUser = {
-    atoms: 0,
+    atoms: 2097152,
```

Comment:
can't we use real data here yet?

Reply:
Not yet. We can put it in mockServer.js though, @mappum ?